### PR TITLE
Mongodb setup

### LINKED ISSUE #9: Connect project to database
Should create weather model and test which validates database operation.

### PR DESCRIPTION
**Pull request by yeareater**
Create weather schema and test insert/retrieve.
This resolves #9.

### CODE REVIEW

**Review by DonghyunNa — APPROVED** [review]

File: tests/mongodb.spec.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  afterEach(function() {
+    if (this.connection != 'undefined') {
+      dailyWeathers.collection.drop();
+    }
+  });
+
+  after(function() {
+    if (this.connection != 'undefined') {
+      this.connection.connection.db.dropDatabase(function() {
+        console.log("dropped.");
```
Comment (crowjdh) [inline tests/mongodb.spec.js]: I forgot to remove this log which looks unnecessary. Maybe we can omit callback function entirely.

File: models/weatherSchema.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const weatherSchema = mongoose.Schema({
+  minTem : String,
+  maxTem : String,
```
Comment (crowjdh) [inline models/weatherSchema.js]: I wonder why these two fields are declaired as string. Can you give me some explanation?

File: tests/mongodb.spec.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+const should = require('should');
+const mongoose = require('mongoose');
+const dailyWeathers = require("../models/weatherSchema");
```
Comment (crowjdh) [inline tests/mongodb.spec.js]: I think the name of the model(class) should start with capital letter.
Check out the link below and consider changing it :)

Reference: [Mongoose Schemas v4.11.5](http://mongoosejs.com/docs/guide.html)

File: tests/mongodb.spec.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  it("should be able to connect to db", function(done) {
+    // TODO: Refactor dependency
+    this.connection = mongoose.connect('mongodb://localhost:27017/worktest', function(err, db) {
+      if (err) {
+        should.exist(null);
+        done();
+      } else {
+        should.exist(true);
+        done();
+      }
```
Comment (crowjdh) [inline tests/mongodb.spec.js]: How about refactoring if-else statement into something like below:

```javascript
// Below is psudocode, so you should check javascript syntax and run test.
should.not.exist(err);
done();
```

File: tests/mongodb.spec.js (hunk 3 above)
Comment (crowjdh) [reply tests/mongodb.spec.js]: Also, I think the name of the class should be "Weather", whichis more general then "DailyWeather".

**Review by crowjdh — CHANGES_REQUESTED** [review]
I left some issues(including suggestions). Please check and consider updating :)

File: models/weatherSchema.js (hunk 2 above)
Comment (yeareater) [reply models/weatherSchema.js]: I thought the weather data includes strings like ℃ or m/s or etc.
I checked it doesn't.

**Review by crowjdh — APPROVED** [review]
Great work!

**Pull request merged**